test/index-test.js

```diff
@@ -146,7 +150,10 @@ module.exports = (repo) => {
       })
     })
 
-    it('block is retrived from peer', (done) => {
+    // Not sure if I understand what is going on here
+    // test fails because now the network is not properly mocked
+    // what are these net.stores and mockNet.bitswaps?
```

Comment:
@dignifiedquire could you clarify what is going on in this test? libp2p needs to be properly mocked, but I couldn't get what was exactly was going on/intended.